src/analysisd/decoders/syscheck.c

```diff
@@ -1194,6 +1203,8 @@ static int fim_process_alert(_sdb * sdb, Eventinfo *lf, cJSON * event) {
                 os_strdup(object->valuestring, lf->fields[FIM_FILE].value);
             } else if (strcmp(object->string, "mode") == 0) {
                 mode = object->valuestring;
+                os_strdup(mode, lf->mode);
```

Comment:
Since we have the mode field inside the `lf` event we can remove the parameter from the `fim_generate_alert(lf, mode, event_type, attributes, old_attributes, audit) function;`

Reply:
Done 612cc75